misc/helm-charts/operator/templates/materialize-environments.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+{{- if .Values.materializeEnvironments }}
+{{- range .Values.materializeEnvironments }}
+---
+apiVersion: materialize.cloud/v1alpha1
+kind: Materialize
+metadata:
+  name: {{ .name }}
+spec:
+  organizationId: {{ .organizationId }}
+  environmentdImageRef: {{ .environmentdImageRef }}
+  externalIdPrefix: {{ .externalIdPrefix }}
+  hostname: {{ .hostname }}
```

Comment:
as a heads up, several of these params were removed: https://github.com/MaterializeInc/materialize/pull/29922/files#diff-20157b422e654d96fd44bce75440eacd9aed80b4abb7c3fa541b9f47b6178d52R42